packages/engine/src/output/utils.rs

```diff
@@ -64,11 +63,4 @@ fn remove_experiment_parts(experiment_id: &ExperimentId) {
             );
         }
     }
-
-    match std::fs::remove_dir(RELATIVE_PARTS_FOLDER) {
```

Comment:
Can we have a comment, that we might want to move the parts-folder to a temporary folder (like `/tmp` or `/var/tmp`)? @Alfred-Mountfield thoughts?

Reply:
Added a comment https://github.com/hashintel/hash/pull/494/commits/683389f6efb7d1e44382dc79b96998088a81d87a